wandb: Agent Starting Run: 5ji42f8i with config:
wandb: 	actor_learning_rate: 3.904474686688955e-05
wandb: 	batch_size: 128
wandb: 	critic_learning_rate: 0
wandb: 	early_stopping_patience: 100
wandb: 	epochs: 800
wandb: 	epsilon: 0.8318673552537419
wandb: 	hdim: 8
wandb: 	learning_rate: 1e-06
wandb: 	reg_coef: 0.43910624946179433
wandb: Currently logged in as: ninabraakman (ninabraakman-university-of-amsterdam) to https://api.wandb.ai. Use `wandb login --relogin` to force relogin
wandb: Tracking run with wandb version 0.19.11
wandb: Run data is saved locally in /gpfs/work5/0/prjs1491/Attention-based-RL-MIL/wandb/run-20250530_015111-5ji42f8i
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run fanciful-sweep-1
wandb: ‚≠êÔ∏è View project at https://wandb.ai/ninabraakman-university-of-amsterdam/MasterThesis
wandb: üßπ View sweep at https://wandb.ai/ninabraakman-university-of-amsterdam/MasterThesis/sweeps/xl8kglob
wandb: üöÄ View run at https://wandb.ai/ninabraakman-university-of-amsterdam/MasterThesis/runs/5ji42f8i
/gpfs/work5/0/prjs1491/Attention-based-RL-MIL/venv/lib64/python3.9/site-packages/networkx/utils/backends.py:135: RuntimeWarning: networkx backend defined more than once: nx-loopback
  backends.update(_get_backends("networkx.backends"))
wandb:                                                                                
wandb: üöÄ View run fanciful-sweep-1 at: https://wandb.ai/ninabraakman-university-of-amsterdam/MasterThesis/runs/5ji42f8i
wandb: ‚≠êÔ∏è View project at: https://wandb.ai/ninabraakman-university-of-amsterdam/MasterThesis
wandb: Synced 5 W&B file(s), 0 media file(s), 0 artifact file(s) and 0 other file(s)
wandb: Find logs at: ./wandb/run-20250530_015111-5ji42f8i/logs
wandb: ERROR Run 5ji42f8i errored:
wandb: ERROR Traceback (most recent call last):
wandb: ERROR   File "/gpfs/work5/0/prjs1491/Attention-based-RL-MIL/venv/lib64/python3.9/site-packages/wandb/agents/pyagent.py", line 306, in _run_job
wandb: ERROR     self._function()
wandb: ERROR   File "/gpfs/work5/0/prjs1491/Attention-based-RL-MIL/run_rlmil.py", line 557, in main_sweep
wandb: ERROR     policy_network = train(
wandb: ERROR   File "/gpfs/work5/0/prjs1491/Attention-based-RL-MIL/run_rlmil.py", line 357, in train
wandb: ERROR     log_dict = get_first_batch_info(policy_network, eval_dataloader, device, bag_size, sample_algorithm)
wandb: ERROR   File "/gpfs/work5/0/prjs1491/Attention-based-RL-MIL/run_rlmil.py", line 310, in get_first_batch_info
wandb: ERROR     action, _ = sample_action(action_probs, bag_size, device, random=False, algorithm=sample_algorithm)
wandb: ERROR   File "/gpfs/work5/0/prjs1491/Attention-based-RL-MIL/models.py", line 540, in sample_action
wandb: ERROR     return sample_static_action(action_probs, n, device, random=random)
wandb: ERROR   File "/gpfs/work5/0/prjs1491/Attention-based-RL-MIL/models.py", line 588, in sample_static_action
wandb: ERROR     action = action_sort.indices[:, :n]
wandb: ERROR IndexError: too many indices for tensor of dimension 1
wandb: ERROR 
wandb: Sweep Agent: Waiting for job.
wandb: Sweep Agent: Exiting.
